tests/sentry/integrations/vsts/test_issues.py

```diff
@@ -164,6 +167,55 @@ def test_sync_assignee_outbound(self):
         assert responses.calls[1].request.url == 'https://fabrikam-fiber-inc.visualstudio.com/_apis/wit/workitems/%d' % vsts_work_item_id
         assert responses.calls[1].response.status_code == 200
 
+    @responses.activate
+    def test_sync_assignee_outbound_with_paging(self):
+        vsts_work_item_id = 5
+        responses.add(
+            responses.PATCH,
+            'https://fabrikam-fiber-inc.visualstudio.com/_apis/wit/workitems/%d' % vsts_work_item_id,
+            body=WORK_ITEM_RESPONSE,
```

Comment:
Is it useful to have this be a callback stub that asserts the request payload so that you know the right user was matched up?

Reply:
yikes... didn't realize I didn't check that! Thanks.